coding-standards-and-naming.rst

```diff
@@ -381,19 +381,8 @@ Extensions SHOULD be well tested using ``*.phpt`` tests. Read more at
 Experimental Functions
 ======================
 
-To reduce the problems normally associated with the first public implementation
-of a new set of functions, it has been suggested that the first implementation
-SHOULD place a file labeled `EXPERIMENTAL` in the function directory. Functions
-MUST follow the standard prefixing conventions during their initial
-implementation.
-
-The file labelled ``EXPERIMENTAL`` SHOULD include the following information:
-
--  Any authoring information (known bugs, future directions of the module).
--  Ongoing status notes which may not be appropriate for Git comments.
-
-In general, new features MUST be in a separate extension, or experimental
-branches, until an RFC is passed to add them to the core distribution.
+New extensions MUST start out as third-party extensions, or in an experimental
+branch, until an RFC is passed to add them to the core distribution.
```

Comment:
I would scrap the whole section as it's just given that you cannot introduce a new extension without the RFC so the rest are just implications that don't need to be specified here.